rules/aws/impact_iam_deactivate_mfa_device.toml

```diff
@@ -36,7 +36,7 @@ timestamp_override = "event.ingested"
 type = "query"
 
 query = '''
-event.dataset:aws.cloudtrail and event.provider:iam.amazonaws.com and event.action:DeactivateMFADevice and event.outcome:success
+event.action:(DeactivateMFADevice or DeleteVirtualMFADevice) and event.dataset:aws.cloudtrail and event.provider:iam.amazonaws.com and event.outcome:success
```

Comment:
```suggestion
event.dataset:aws.cloudtrail and event.provider:iam.amazonaws.com and event.action:(DeactivateMFADevice or DeleteVirtualMFADevice) and event.outcome:success
```

Reply:
lets keep the same order of ops as the other AWS rules.

Reply:
Roger.